lib/components/MusicScreen/music_screen_tab_view.dart

```diff
@@ -415,6 +414,11 @@ class _MusicScreenTabViewState extends State<MusicScreenTabView>
   }
 }
 
+class MusicRefreshCallback {
+  void call() => callback?.call();
+  void Function()? callback;
+}
+
```

Comment:
Is there a particular reason why you used a class instead of just a regular function?

Reply:
It’s because the function is basically being passed backwards.  The callback is called by the parent, but needs to be set by the child.  I think I can do the same thing by passing down the map, though.